roles/security/tasks/main.yml

```diff
@@ -22,35 +23,51 @@
               regexp="{{ server_user }} ALL"
               line="{{ server_user }} ALL=(ALL) ALL"
               state=present
+  when: server_user != "root"
 
-- name: Install required packages
-  apt: update_cache={{ update_apt_cache }} state=installed pkg={{ item }}
-  with_items: 
-  - "{{ required_packages }}"
+# Unattended Upgrades settings
```

Comment:
@jcalazan I wasn't sure of a way to continue looping over the `required_packages` variable while also making each package optional. Therefore I am installing each packages separately in the `tasks` section. If you have a better solution let me know?